cmd/manager/main.go

```diff
@@ -53,15 +53,25 @@ func main() {
 	}
 
 	// Become the leader before proceeding
-	leader.Become(context.TODO(), "devconsole-lock")
+	err = leader.Become(context.TODO(), "devconsole-lock")
+	if err != nil {
+		log.Error(err, "")
```

Comment:
Don't we have a `log.Panic` maybe?

Reply:
I only updated this to make it compliant to the linter.